src/contracts/core/DelegationManager.sol

```diff
@@ -206,20 +207,33 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         
         if (!stakerCanUndelegate(staker)) {
             // force the staker into "undelegation limbo" in the EigenPodManager if necessary
-            eigenPodManager.forceIntoUndelegationLimbo(staker);
+            uint256 podShares = eigenPodManager.forceIntoUndelegationLimbo(staker, operator);
+
+            IStrategy[] memory strategies;
+            uint256[] memory strategyShares;
 
             // force a withdrawal of all of the staker's shares from the StrategyManager
-            queuedWithdrawal = strategyManager.forceTotalWithdrawal(staker);
+            (strategies, strategyShares, queuedWithdrawal)
```

Comment:
weird syntax here, can we declare these in the call to SM?

Reply:
also, this should be called `withdrawalRoot` and an `@returns` should be added

Reply:
> also, this should be called `withdrawalRoot` and an `@returns` should be added

there is a `@return` tag, but I'll correct the name.
Solidity doesn't appear to like assigning to a mix of newly-declared params and already-declared params in a single call, which is why I have that uglier syntax. 
I tried something like 
```
            (IStrategy[] memory strategies, uint256[] memory strategyShares, withdrawalRoot)
                = strategyManager.forceTotalWithdrawal(staker);
```
but the compiler throws an error:
```
Compiler run failed:
Error (2314): Expected identifier but got ')'
   --> src/contracts/core/DelegationManager.sol:216:92:
    |
216 |             (IStrategy[] memory strategies, uint256[] memory strategyShares, withdrawalRoot)
```

Reply:
naming changed in commit https://github.com/Layr-Labs/eigenlayer-contracts/pull/183/commits/bf1de8a06817eb639c1c5a265b1aedfe0c9e1a33
let me know if you can sort out making the cleaner syntax work correctly?